connect-rest-extension/pom.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <parent>
+    <groupId>io.debezium</groupId>
+    <artifactId>debezium-parent</artifactId>
+    <version>1.7.0-SNAPSHOT</version>
+    <relativePath>../debezium-parent/pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>debezium-connect-rest-extension</artifactId>
+  <name>Debezium Kafka Connect REST Extension</name>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
Most, if not all, of that info should be provided via the parent POM and BOM POM.

Reply:
Could be a duplicate because this pom was moved from the other repo where it was an independent project. I will double-check.